### DISCUSSION TITLE
Plugin Site Unavailable

### DISCUSSION THREAD

**Discussion by photografia** [homebridge-simplisafe3/homebridge-simplisafe3#376] [General]
When trying to install in hoobs, using the UI and the CLI, I get the error "Plugin Site Unavailable".

Any idea why this is happening?

**Comment by photografia** [comment]
hoobs@hoobs:~$ npm install -g --unsafe-perm homebridge-simplisafe3
npm ERR! code EAI_AGAIN
npm ERR! syscall getaddrinfo
npm ERR! errno EAI_AGAIN
npm ERR! request to https://registry.npmjs.org/homebridge-simplisafe3 failed, reason: getaddrinfo EAI_AGAIN registry.npmjs.org

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/hoobs/.npm/_logs/2022-12-13T16_24_39_475Z-debug-0.log
hoobs@hoobs:~$ npm install homebridge-simplisafe3
npm ERR! code EAI_AGAIN
npm ERR! syscall getaddrinfo
npm ERR! errno EAI_AGAIN
npm ERR! request to https://registry.npmjs.org/homebridge-simplisafe3 failed, reason: getaddrinfo EAI_AGAIN registry.npmjs.org

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/hoobs/.npm/_logs/2022-12-13T16_26_53_220Z-debug-0.log

**Comment by shamoon** [comment]
Either the internet isn’t working from your container or it’s a question for HOOBS folks, no issue with the plug-in itself

**Comment by photografia** [comment]
Thnks for looking. From Hoobs Q&A:

A recent Linux system update caused some DNS issues affecting HOOBS Box users running on model HSLF-1 as well as DIY installs on Raspberry Pi boards.
If you are not able to install plugins or your plugins fail to connect to the internet after a System update, and your logs read "Plugin site unavailable",
this likely affects you. To fix the issue, you need to remove the resolv.conf file and reboot your system using the following terminal command:

sudo rm /etc/resolv.conf; sudo reboot

**Reply by Homekitdude** [reply]
I have this issue but this is the reason why I never wanted to get into homebridge stuff. None of my stuff works all the sudden and I have no idea how to fix it. When I try to login to terminal it asks for my terminal password which I do not know. I'm about to abandon this whole platform, so bummed.

**Reply by photografia** [reply]
Login with the default username and password:

Username: admin
Password: admin
